public/src/components/channelManagement/headTests/headTestVariantEditorSecondaryCtaEditor.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
this file looks the same as `variantEditorCtaEditor.ts`

Reply:
This new file is a copy of `../variantEditorCtaEditor.ts` with just the interface name changed. I ran into some compile issues around primary and secondary CTA types (they differ in banners) which got solved when I made a local copy of the variantEditor file for the header secondaryCTA buttons. I'm not proud of the change but it fixed the types issue; I'm happy to get rid of this fix if you can help me solve those type errors.

Reply:
ah I see. I'll have a think...

Reply:
it looks like this is no longer an issue, I just swapped `HeadTestVariantEditorSecondaryCtaEditor` for `VariantEditorCtaEditor` and it works fine, so I think you can delete this file.

Reply:
I'll give it a go

Reply:
Done